### PR TITLE
BATS: k8s: fall over if we need an IP but it's unset

### PR DESCRIPTION
**Pull request by mook-as**
This is mostly so that we can get better failure messages.

### CODE REVIEW

File: bats/tests/k8s/helm-install-rancher.bats [hunk 1]
```diff
@@ -35,7 +36,7 @@ deploy_rancher() {
         --install rancher rancher-latest/rancher \
         --version "${RD_RANCHER_IMAGE_TAG#v}" \
         --namespace cattle-system \
-        --set hostname="$(get_host)" \
+        --set hostname="$(get_host || return)" \
```
Comment (jandubois) [inline bats/tests/k8s/helm-install-rancher.bats:39]: I don't like the `return` nested within parameter substitution inside a big block of parameters. If this is necessary, then I would prefer something like:

```bash
local host
host=$(get_host) || return
helm upgrade … \\
    --set hostname="$host"
```

We really need to figure out how to enable `errexit` inside `run`, to get rid of this silliness.

**Review by jandubois — CHANGES_REQUESTED** [review]
Purely stylistic change request

File: bats/tests/k8s/helm-install-rancher.bats (hunk 1 above)
Comment (mook-as) [reply bats/tests/k8s/helm-install-rancher.bats]: Changed. (Also accidentally pushed this branch to upstream; deleted it.)

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Pull request merged**